lib/__tests__/ignore.test.js

```diff
@@ -6,467 +6,341 @@ const standalone = require('../standalone');
 
 const fixturesPath = path.join(__dirname, 'fixtures');
 
-describe('extending config and ignoreFiles glob ignoring single glob', () => {
-	let results;
-
-	beforeEach(() => {
-		return standalone({
-			files: [`${fixturesPath}/empty-block.css`, `${fixturesPath}/invalid-hex.css`],
-			config: {
-				ignoreFiles: '**/invalid-hex.css',
-				extends: ['./config-block-no-empty', './config-color-no-invalid-hex'],
-			},
-			configBasedir: path.join(__dirname, 'fixtures'),
-		}).then((data) => (results = data.results));
-	});
-
-	it('two files found', () => {
+test('extending config and ignoreFiles glob ignoring single glob', () => {
+	return standalone({
+		files: [`${fixturesPath}/empty-block.css`, `${fixturesPath}/invalid-hex.css`],
+		config: {
+			ignoreFiles: '**/invalid-hex.css',
+			extends: ['./config-block-no-empty', './config-color-no-invalid-hex'],
+		},
+		configBasedir: path.join(__dirname, 'fixtures'),
+	}).then(({ results }) => {
+		// two files found
 		expect(results).toHaveLength(2);
-	});
 
-	it('empty-block.css found', () => {
+		// empty-block.css found
 		expect(results[0].source.indexOf('empty-block.css')).not.toBe(-1);
```

Comment:
Don’t we need to check here that invalid-hex.css possible here instead?

Reply:
I think we have it here to be sure that the first result is for file we expect.